### ISSUE TITLE
"Calling the API" section of "Getting started" is hard to follow

### ISSUE THREAD

**Issue by RPrimrose** [IdentityServer/Documentation#12]
I found the "Calling the API" section somewhat hard to follow, I think changing the following points would make it easier to read:
* When you're told to install the "Thinktecture.IdentityModel.Client" package it's not clear which of the two projects you should install it to. Specifying this and adding a little bit of context around what the package does would be great.

* The code snippets for GetTokenAsync and CallApi are introduced before there's any mention of them being contained in a new controller. This lack of context is confusing to the reader and if you're writing code from the tutorial as you go you don't know what class to put them in. It'd be better to describe the new controller first and then detail the two helper methods.

**Comment by leastprivilege** [comment]
Hey, 

thanks for all the feedback - I am in the middle of transferring the data to the new doc website

the walkthrough is here:
https://thinktecture.github.io/Thinktecture.IdentityServer.v3.Documentation/docs/overview/mvcGettingStarted.html

when you scroll all the way down - there is "edit page" link (will be moved to the beginning of the page) - please make the fixes yourself - but do them one at a time so it is easier for me to review them.

thanks!

**Comment by leastprivilege** [comment]
Are you going to make these changes yourself or should I do that?
